static/app/views/issueDetails/traceTimeline/traceTimeline.tsx

```diff
@@ -49,28 +64,58 @@ export function TraceTimeline({event}: TraceTimelineProps) {
 
   return (
     <ErrorBoundary mini>
-      <TimelineWrapper>
-        <div ref={timelineRef}>
-          <TimelineEventsContainer>
-            <TimelineOutline />
-            {/* Sets a min width of 200 for testing */}
-            <TraceTimelineEvents event={event} width={Math.max(width, 200)} />
-          </TimelineEventsContainer>
-        </div>
-        <QuestionTooltipWrapper>
-          <QuestionTooltip
-            size="sm"
-            title={t(
-              'This is a trace timeline showing all related events happening upstream and downstream of this event'
-            )}
-            position="bottom"
-          />
-        </QuestionTooltipWrapper>
-      </TimelineWrapper>
+      {timelineStatus === 'shown' ? (
+        <TimelineWrapper>
+          <div ref={timelineRef}>
+            <TimelineEventsContainer>
+              <TimelineOutline />
+              {/* Sets a min width of 200 for testing */}
+              <TraceTimelineEvents event={event} width={Math.max(width, 200)} />
+            </TimelineEventsContainer>
+          </div>
+          <QuestionTooltipWrapper>
+            <QuestionTooltip
+              size="sm"
+              title={t(
+                'This is a trace timeline showing all related events happening upstream and downstream of this event'
+              )}
+              position="bottom"
+            />
+          </QuestionTooltipWrapper>
+        </TimelineWrapper>
+      ) : (
+        <Feature features="related-issues-issue-details-page">
+          {timelineSkipped && (
+            // XXX: Temporary. This will need to be replaced with a styled component
+            <div style={{width: '400px'}}>
+              {traceEvents.map((traceEvent, index) => (
+                <div key={index} style={{display: 'flex', alignItems: 'center'}}>
+                  <div
+                    style={{whiteSpace: 'nowrap', minWidth: '75px'}}
+                    data-test-id={`this-event-${traceEvent.id}`}
+                  >
+                    {event.id === traceEvent.id && <span>This event</span>}
```

Comment:
@malwilley from yesterday, I added the following feature of showing "This event". Would you test more than the following?

```javascript
    let element = await screen.getByTestId('this-event-1');
    expect(element).toHaveTextContent('This event');
    element = await screen.getByTestId('this-event-abc');
    expect(element).toHaveTextContent('');
```

First screenshot:
<hr />
<img width="418" alt="image" src="https://github.com/getsentry/sentry/assets/44410/6c0f932c-a112-4541-b832-dacb792951ff">

Loading the second event will take us to the associated event and do the opposite ordering:
<hr />
<img width="405" alt="image" src="https://github.com/getsentry/sentry/assets/44410/d5a4f1fe-8b4a-430d-b60e-9cb37f8ad3ac">

Reply:
That looks like a good test to me!